test/conftest.py

```diff
@@ -132,17 +132,23 @@ def mocked_channels_list(self, channel_type='channels', **kwargs):
         cursor = kwargs.get('cursor', 0)
         limit = kwargs.get('limit', 100)
 
+        def is_channel_type(channel, channel_type):
```

Comment:
`is_channel_type = partial(dict.get, channel, default=False)`

Reply:
ye nice

Reply:
Nah doesn't work

Reply:
```python
>>> from functools import partial
>>> x = {}
>>> fn = partial(dict.get, x, default=False)
>>> fn(43)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: get() takes no keyword arguments
>>> fn(43)
```

Reply:
kek. good to merge then?